pkg/lib/patternfly/patternfly-4-overrides.scss

```diff
@@ -76,3 +76,8 @@ svg {
       }
   }
 }
+
+// PF assumes check sizes are Chrome-based; size needs to be set for cross-browser compatibility
```

Comment:
Right as you mentioned already, @garrett this will need an upstream issue and the l link here in the code :)

Reply:
Yep, currently working on making sure there's a solution that works across browsers and can be upstreamed to PF. What's in this PR isn't that (yet).

I have something that seems to work correctly across all browsers now; I just need to convert it back to PF variables and ensure that works too.